
Included e.g. during net-tools compilation.

  - Rene Rebe <rene@exactcode.de>

--- ./include/asm-arm/param.h.orig	2005-02-07 13:10:49.040076752 +0100
+++ ./include/asm-arm/param.h	2005-02-07 13:10:56.845890088 +0100
@@ -10,8 +10,6 @@
 #ifndef __ASM_PARAM_H
 #define __ASM_PARAM_H
 
-#include <asm/arch/param.h>	/* for HZ */
-
 #ifndef __KERNEL_HZ
 #define __KERNEL_HZ	100
 #endif
